migrations/versions/d25d457860de_lowercase_discount_code.py

```diff
@@ -0,0 +1,30 @@
+"""lowercase discount code
+
+Revision ID: d25d457860de
+Revises: 6f7b6fad3f54
+Create Date: 2019-05-19 02:06:14.730180
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = 'd25d457860de'
+down_revision = '6f7b6fad3f54'
+
+
+def upgrade():
+    op.execute("UPDATE discount_codes SET code = concat(code, '_') where code not in (SELECT DISTINCT ON (upper(code)) code FROM discount_codes group by event_id, code);",
```

Comment:
Where's the code of changing the column type to citext?

Reply:
@iamareebjamal Sorry. will update.